Modules/ThirdParty/Expat/src/expat/CMakeLists.txt

```diff
@@ -178,6 +178,11 @@ CONFIGURE_FILE(${ITK3P_EXPAT_SOURCE_DIR}/expatDllConfig.h.in
 
 configure_file(expat_config.h.cmake "${ITK3P_EXPAT_BINARY_DIR}/expat_config.h")
 
+set(MANGLE_PREFIX itk_expat)
```

Comment:
Does this respect cache? If the calling code has `set(MANGLE_PREFIX slicer_expat)`, will this line not override that?

Reply:
Purpose of these changes it to introduce a uniform mangling interface for all third-parties.